src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagJobs.java

```diff
@@ -29,10 +29,6 @@
      * Schedule each Bugsnag job based on the delay configuration parameters
      */
     public static void initialize() {
-        if (BugsnagDispatcher.BUGSNAG_ORGANIZATION == null) {
-            LOG.error("WARNING: Bugsnag organization is not available. Cannot schedule bugsnag jobs.");
-            return;
-        }
```

Comment:
I think we may want to keep this optional actually. I can see some cases we don't want Bugsnag running on middleware (e.g., an instance that is just designed to provide extra features to OTP, but not all of the error event stuff).

Reply:
Updated.